backend/internal/models/media.go

```diff
@@ -28,11 +28,13 @@ type Album struct {
 }
 
 type Track struct {
-	Media    MediaType
-	ID       int    `json:"id"`
-	AlbumID  int    `json:"album_id"`
-	Title    string `json:"title"`
-	Duration int    `json:"duration_seconds"`
+	Media       MediaType
```

Comment:
```suggestion
	Media       MediaType `json:"media"`
```
super small but can we give this a new json tag to match the rest